climada/hazard/test/test_tc_tracks.py

```diff
@@ -283,6 +284,23 @@ def test_read_with_basin(self):
         for tr in tc_track.data:
             self.assertEqual(tr.basin[0], "EP")
             self.assertIn("WP", tr.basin)
+    
+    def test_discard_single_points(self):
+        """Check discard_single_points option"""
+        tc_track_singlept = tc.TCTracks()
+        passed = False
+        for year in range(1951, 1981):
```

Comment:
This is a bit of a weird loop. It his to a)  take care of changing IBTrACS data and b) being fast?

Reply:
Yes I agree it's weird, but exactly for the reasons you assume. Currently there is a single-point track in 1951 but if this changes then it would try further years. If none of all these 30 years contain single-point tracks at some point then the test could be updated by taking earlier years.

Reply:
Why not let the loop run over more years then just to be sure?

Reply:
Ok I'll start at 1863, which also has a unique points track, as well as 1865, 1866, 1867 and many others.